pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+markers =
+    last: marker supplied by but not registered by pytest-ordering module
```

Comment:
pytest configuration is already in `tox.ini`. I think your overshadowing that by creating this file.

Reply:
I was getting a warning running the build; adding this file fixed the warning.

Reply:
I'm just saying you should just add the “markers” setting to the respective section in *tox.ini* instead to avoid the extra file and because I think pytest will ignore the other configuration already in *tox.ini* if you create this one.[citation needed]